doc/rados/operations/bluestore-migration.rst

```diff
@@ -114,7 +114,8 @@ Whole host replacement
 If you have a spare host in the cluster, or have sufficient free space
 to evacuate an entire host in order to use it as a spare, then the
 conversion can be done on a host-by-host basis with each stored copy of
-the data migrating only once.
+the data migrating only once.  To evacuate an existing host, simply mark 
+all of its OSDs out and let the cluster rebalance.
```

Comment:
With this method you only (maybe) need to evacuate one host, at the beginning, in order to get the spare empty one for the first swap.  I would move this comment down to the bit about identifying an empty host so that it's clear it's not needed for each step?

Reply:
Actually, this should just be dropped.  Preparing the empty host involves taking it out of the hierarchy, so marking out isn't needed.  We can add a separate paragraph there about how to use an existing host as the spare (assuming there is enough room etc).

After number 2, for example, we can add a second paragraph:

If you would like to use an existing host that is already part of the cluster, and there is sufficient free space on that host so that all of its data can be migrated off, then you can instead do:
<pre>
ceph osd crush unlink $NEWHOST default
</pre>
where default is the immediately ancestor in the CRUSH map.  (For smaller clusters with unmodified configurations this will normally be "default", but it might also be a rack name.)  This will move the host out of the CRUSH hierarchy and cause all data to be migrated off.  Once it is completely empty of data, you can proceed:
<pre>
while ! ceph osd safe-to-destroy $(ceph osd ls-tree $NEWHOST); do sleep 60 ; done
</pre>
(Also, let's merge #17011 first)

Reply:
I made changes as suggested.  Just copied and pasted w/ formatting and `s/immediately/immediate` in the last paragraph.